packages/parser/src/language/architecture/architecture.langium

```diff
@@ -20,31 +19,35 @@ fragment Statement:
 ;
 
 fragment LeftPort:
-    ':'lhsDir=ARROW_DIRECTION
+    ':'? lhsDir=ARROW_DIRECTION
```

Comment:
See above comment on making this optional

Reply:
As making the colon optional is unrelated to how strings are handled, it should not be included in this PR. 

---

As to making it optional, I don't think we should do it, as it introduces unnecessary ambiguity. 

The colon was suggested by @knsv in https://github.com/mermaid-js/mermaid/pull/5452#issuecomment-2245285669 to align the syntax to other diagrams.